components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/SessionNonceCookieUtil.java

```diff
@@ -136,4 +136,15 @@ public static boolean isNonceCookieEnabled() {
         return nonceCookieConfig;
     }
 
+    /**
+     * Check if nonce cookie validation should be skipped based on the request.
+     *
+     * @param request Http servlet request.
+     * @return True if nonce cookie validation should be skipped.
+     */
+    public static boolean skipNonceCookieValidation(HttpServletRequest request) {
```

Comment:
use some name like validateNonceCookie to avoid double negation

Reply:
wouldn't a method name like validateNonceCookie give the notion that the method performs the validation. thought of renaming to `isNonceCookieValidationSkipped `